src/main/java/org/mamute/dao/QuestionDAO.java

```diff
@@ -278,5 +277,126 @@ public Question questionWith(Attachment attachment) {
 				.setParameter("a", attachment)
 				.uniqueResult();
 	}
-}
 
+	public void delete(Question question) {
```

Comment:
I don't think we have too many cases like that, can u grep for `createSQLQuery` just for us to get an idea? I'm not with my notebook now